prost_build/src/main.rs

```diff
@@ -21,15 +21,15 @@ const EXCLUDED_PROTO_PACKAGES: &[&str] = &["gogoproto", "google", "tendermint",
 const COSMOS_SDK_PROTO_REGEX: &str = "(super::)+cosmos";
 
 /// the output directory
+const TMP_PATH: &str = "/tmp/sommelier";
```

Comment:
Why move to a hardcoded path rather than using the environment's tempdir?

Reply:
It was causing weird file permissions errors and this is the way other proto builds do it in our repos